DNSAuth/config.go

```diff
@@ -2,15 +2,13 @@ package main
 
 import (
 	"github.com/shafreeck/configo"
-	"github.com/Packet-Clearing-House/DNSAuth/DNSAuth/bgp"
 )
 
 type Config struct {
-	BGP *bgp.BGPConf
-	CustomerDB string `cfg:"customer-db; required; "`
-	CustomerRefresh int `cfg:"customer-refresh; 24; "`
-	InfluxDB string `cfg:"influx-db; required; "`
-	WatchDir string `cfg:"watch-dir; required; "`
+	CustomerDB      string `cfg:"customer-db; required; "`
```

Comment:
No biggie, but try and keep formatting changes to at least their own commit, if not their own PR like you did in #22 ;)

Reply:
Oh! I see you said, "Note: some whitespace and linting also made it through.".  While good to call, still better to do in a separate commit - most important - still no biggie ;)

Reply:
:+1: